apis/Google.Cloud.BigQuery.V2/Google.Cloud.BigQuery.V2/BigQueryDataset.cs

```diff
@@ -292,10 +292,53 @@ public BigQueryDataset Patch(Dataset resource, bool matchEtag, PatchDatasetOptio
         /// Deletes this dataset.
         /// This method just creates a <see cref="DatasetReference"/> and delegates to <see cref="BigQueryClient.DeleteDatasetAsync(string, string, DeleteDatasetOptions, CancellationToken)"/>.
         /// </summary>
-        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
         /// <param name="options">The options for the operation. May be null, in which case defaults will be supplied.</param>
+        /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
         /// <returns>A task representing the asynchronous operation.</returns>
         public Task DeleteAsync(DeleteDatasetOptions options = null, CancellationToken cancellationToken = default(CancellationToken)) =>
             _client.DeleteDatasetAsync(Reference, options, cancellationToken);
+
+        /// <summary>
+        /// Asynchronously updates this dataset to match the specified resource.
+        /// </summary>
+        /// <remarks>
+        /// This method delegates to <see cref="BigQueryClient.UpdateDataset(DatasetReference, Dataset, UpdateDatasetOptions)"/>.
```

Comment:
Well caught. That wasn't on my personal checklist! Will fix. Probably wrong on the BigQueryClient methods as well - or I'm not documenting the delegation in the way that I should...